docs/config/index.md

```diff
@@ -366,18 +366,45 @@ export default defineConfig({
 })
 ```
 
-To get TypeScript working with the global APIs, add `vitest/globals` to the `types` field in your `tsconfig.json`
+#### TypeScript integration
 
-```json
-// tsconfig.json
-{
-  "compilerOptions": {
-    "types": ["vitest/globals"]
-  }
-}
-```
+To register global API types, follow one of the following methods:
+
+1. Use a `@types/vitest.d.ts` file
+
+    Create a `@types/vitest.d.ts` file in your project root that imports `vitest/globals`:
+
+    ```ts
+    import 'vitest/globals'
+    ```
```

Comment:
In my experiments, this is the best method because it's clear, doesn't require any hack like `import type {}`, doesn't pollute source code, and works even if `types`/`typeRoots` are overridden.

Reply:
Maybe we should also mention it here in the docs?